.github/workflows/create-deploy-release.yml

```diff
@@ -0,0 +1,46 @@
+name: Create and deploy a new release
+run-name: create and deploy ${{ github.sha }}
+on:
+  push:
+    branches:
+      - main
+jobs:
+  trigger-gitlab-pipeline:
+    runs-on: ubuntu-latest
+    env:
+      # static variables and secrets to be defined
+      APP_NAME: ${{ vars.APP_NAME }}
```

Comment:
These vars should be configured in the repo settings?

Reply:
Yes, and it's already done for this repository.